README.md

```diff
@@ -67,3 +67,37 @@ public pull request.
 
 Pulls the upstream repository into the local repository.
 
+## [`cache_pdks`](./cache_pdks)
+
+Checks if there is a cache for a given PDK install. The cache hash used
+consists of `open_pdks`, `sky130` and `magic` commit hashes. If it can not
+find a cached install, the cache is built. Arguments:
```

Comment:
> the cache is built?

a new PDK is built according to the specified `ol_path` and `ol_tag`.

Reply:
Changed